server/controllers/documents.js

```diff
@@ -0,0 +1,135 @@
+import { Documents } from '../models/';
+
+const documentsController = {
+  create(req, res) {
+    return Documents.create({
+      documentName: req.body.documentName,
```

Comment:
What happens if I try to create a document with `bad records`, is there a way to check for those and send back appropriate response? You may want to use middleware for cases like this because I don't notice one.
Especially for:
CRUD actions on documents, users and roles